roles/openshift_logging/defaults/main.yml

```diff
@@ -72,7 +72,7 @@ openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nod
 openshift_logging_fluentd_cpu_limit: 100m
 openshift_logging_fluentd_memory_limit: 512Mi
 openshift_logging_fluentd_es_copy: false
-openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal | default('') }}"
+openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal if openshift_hosted_logging_use_journal is defined else (docker_log_driver|default('') == 'journald') | ternary(True, False) }}"
```

Comment:
So it looks like we can use `openshift.docker.log_driver` ?

On the other hand, the less we need to rely on `openshift_facts` the better. I would just be worried about `docker_log_driver` being set incorrectly compared to what's actually being used.
I think the risk of that is likely minimal though... so LGTM as is